src/Sylius/Bundle/RbacBundle/spec/Form/Type/PermissionTypeSpec.php

```diff
@@ -46,17 +47,18 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
-            ->add('parent', 'sylius_permission_choice', Argument::any())
+            ->addEventSubscriber(Argument::type(AddCodeFormSubscriber::class))
             ->shouldBeCalled()
```

Comment:
`shouldBeCalled()` should be removed as there is call `willReturn()`, I have described & explained this quite few times already what are differences between those two.